src/containers/shared/components/Transaction/AMM/AMMBid/parser.tsx

```diff
@@ -0,0 +1,20 @@
+import {
```

Comment:
This should be `parser.ts` since it is not a React component

Reply:
ditto for the rest

Reply:
done

Reply:
also renamed the index files to index.ts